greenfoot/src/main/java/greenfoot/vmcomm/VMCommsSimulation.java

```diff
@@ -541,7 +541,7 @@ else if (Command.isMouseEvent(data[0]))
                         break;
                     case Command.COMMAND_END_DRAG:
                         // Will be drag-ID:
-                        WorldHandler.getInstance().finishDrag(data[1]);
+                        WorldHandler.getInstance().finishDrag(data[1], data[2], data[3]);
```

Comment:
I suppose we can add what the 2 new other args will be (or are, really) , in the comment above.